sky/services/platform/interface.mojom

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module platform_interface;
+
+/// Specifies an acceptable orientation for the interface to be in
+/// A mask of acceptable interface orientations is created by or'ing individual
+/// options
+enum /* options */ InterfaceOrientation {
+  Portrait  = 1,
+  Landscape = 2,
+};
+
+/// Specifies a particular device orientation
+/// A mask of device interface orientations is created by or'ing individual
+/// options
+enum /* options*/ DeviceOrientation {
+  PortraitUp      = 1,
+  PortraitDown    = 2,
+  LandscapeLeft   = 4,
+  LandscapeRight  = 8,
+};
+
+/// Specifies how the status bar(s) should be displayed
```

Comment:
like, the Facebook chat bubbles. If we can't disable those then skip it. I
was mostly just trying to show it could be extended.

On Thu, Jan 28, 2016, 17:05 Chinmay Garde <notifications@github.com> wrote:

> In sky/services/platform/interface.mojom
> <https://github.com/flutter/engine/pull/2315#discussion_r51213574>:
>
> > +enum /* options */ InterfaceOrientation {
> > +  Portrait  = 1,
> > +  Landscape = 2,
> > +};
> > +
> > +/// Specifies a particular device orientation
> > +/// A mask of device interface orientations is created by or'ing individual
> > +/// options
> > +enum /* options*/ DeviceOrientation {
> > +  PortraitUp      = 1,
> > +  PortraitDown    = 2,
> > +  LandscapeLeft   = 4,
> > +  LandscapeRight  = 8,
> > +};
> > +
> > +/// Specifies how the status bar(s) should be displayed
>
> What does Floating map to?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/flutter/engine/pull/2315/files#r51213574>.
>